frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -570,6 +570,11 @@ export const CRDCard: React.SFC<CRDCardProps> = (props) => {
     `/k8s/ns/${csv.metadata.namespace}/${ClusterServiceVersionModel.plural}/${
       csv.metadata.name
     }/${reference}/~new`;
+  if (crd.displayName.startsWith("[Internal]")) {
```

Comment:
Is there a reason we can't use the `operators.operatorframework.io/internal-objects` annotation instead of trying to key on the name? See #3518

Reply:
I tried this approach but is not working for me. Is there any documentation that clearly explains how to achieve this?

Reply:
@umangachapagain ^^

Reply:
See https://github.com/operator-framework/operator-lifecycle-manager/pull/1097

Reply:
Keying on name should be removed ?